app/views/schedule_rules/_schedule_rule_fields.html.haml

```diff
@@ -15,10 +15,13 @@
       = f.label :end_time
       = time_select24(f, :end, hours: (0..24), minute_step: @product.reserve_interval)
 
-= f.input :discount_percent, label: false, hint: false do
-  = f.label :discount_percent
-  = f.hint :discount_percent
-  = f.number_field :discount_percent, size: 4
+%h3 Global Price Group Discounts
+
+= f.association :price_group_discounts do |pgd|
+  = pgd.label pgd.object.price_group.name
+  = pgd.hint :discount_percent
```

Comment:
Is this showing up like `"During the period covered by this schedule rule, overall cost will be discounted by"`?  I don't see it in the screenshot

Reply:
We should add a hint here (or maybe just at the top level) indicating that non-global price groups will get the discount based on internal/external discount.